doc/source/user_guide/fields/field_info.rst

```diff
@@ -48,35 +51,30 @@ calling the ``get_scalar_fields_info`` method.
    'absolute-pressure': {'display_name': 'Absolute Pressure', 'section': 'Pressure...', 'domain': 'mixture'},
     ...}
 
-Get range
-~~~~~~~~~
-You can request the range (minimum and maximum values) for a given ``field`` by
-calling the ``get_range`` method. It takes a ``field`` argument which can be obtained
-from the keys of the dictionary returned by ``get_scalar_fields_info`` method.
+Scalar field range
+~~~~~~~~~~~~~~~~~~
+To get the range (minimum and maximum values) of a specific scalar field, use the ``get_scalar_field_range`` method.
+The field name must be one of the keys returned by the ``get_scalar_fields_info`` method.
```

Comment:
All places where methods are mentioned like ``get_scalar_field_range`` should be updated as follows:

:meth:`get_scalar_field_range() <ansys.fluent.core.services.field_data.get_scalar_field_range>`

This helps to create a clickable hyperlink and users can directly navigate to the correct API of the mentioned method.

This will also avoid confusion between 3 API references where `get_scalar_field_range` is available. e.g. field_data_intefaces.py, file_session.py, field_data.py,

Reply:
Thank you @hpohekar. Done.